### DISCUSSION TITLE
No power measurements on BlitzwolfSHP6

### DISCUSSION THREAD

**Discussion by Cyroq** [arendst/Tasmota#11061] [Q&A] [answered]
Hi, I flashed the Blitzwolf SHP6 15A with tasmota.bin. I can access the device via wifi but the measured values (voltage, current, etc) are all 0. Except kwh, that is slowly increasing. So I'd say the sensors are working but somehow the values aren't shown. What could be the problem here?

**Comment by ascillato** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you. 

Please check the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md)

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
